vm/interpreter/src/vm.rs

```diff
@@ -483,6 +484,8 @@ where
             0,
             self.rand,
             &self.registered_actors,
+            circ_supply::setup_preignition_genesis_actors_testnet(self.store).ok(),
+            circ_supply::setup_postignition_genesis_actors_testnet(self.store).ok(),
```

Comment:
You're ignoring the error and just acting as if it was none? this will not be intentional behaviour